src/status_im2/contexts/onboarding/enable_biometrics/view.cljs

```diff
@@ -17,18 +18,38 @@
 
 (defn page
   []
-  [rn/view {:style style/page-container}
-   [navigation-bar]
-   [rn/view {:style {:padding-horizontal 20}}
-    [quo/text
-     {:size   :heading-1
-      :weight :semi-bold
-      :style  {:color colors/white}} "Enable-biometrics"]
-    [quo/button
-     {:on-press       #(rf/dispatch [:onboarding-2/create-account-and-login])
-      :type           :grey
-      :override-theme :dark
-      :style          {}} (i18n/label :t/continue)]]])
+  (let [supported-biometric (rf/sub [:supported-biometric-auth])
+        bio-type-label      (biometric/get-label supported-biometric)
+        profile-color       (:color (rf/sub [:onboarding-2/profile]))]
+    [rn/view {:style style/page-container}
+     [navigation-bar]
+     [rn/view
+      {:style {:padding-horizontal 20
+               :flex               1}}
+      [quo/text
+       {:size   :heading-1
+        :weight :semi-bold
+        :style  {:color colors/white}} (i18n/label :t/enable-biometrics)]
+      [quo/text
+       {:size  :paragraph-1
+        :style {:color      colors/white
+                :margin-top 8}}
+       (i18n/label :t/use-biometrics)]
+      ;; Replace view with image view with the real illustration
```

Comment:
let's create an issue with a TODO - it can even be a generic issue covering all images for onboarding

Reply:
Sure, I'll create the issue and reference here